ubuntudesign/www.ubuntu.com
# add japanese AU service description and tidied up the UA indentation

## Done

 - Added new Japanese service description for UA
 - Fixed issues in files I touched

## QA

 - Go to `http://10.20.66.17:8001/legal/ubuntu-advantage/service-description` click the link in "Japanese translation"
 - Check that the Japanese page displays correctly
 - Check the content against the [copydoc](https://docs.google.com/document/d/1Tpf4FuP6SzvfyFw-nlR44uAIkofNdwzDaJFY36ZAk68/edit#) as much as you are able.
